### PR TITLE
[plg_content_loadmodule]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_content_loadmodule
### Testing Instructions

code review

### CODE REVIEW

File: plugins/content/loadmodule/loadmodule.xml [hunk 1]
```diff
@@ -19,15 +19,38 @@
...
-					label="PLG_LOADMODULE_FIELD_STYLE_LABEL">
-					<option value="table">PLG_LOADMODULE_FIELD_VALUE_TABLE</option>
-					<option value="horz">PLG_LOADMODULE_FIELD_VALUE_HORIZONTAL</option>
-					<option value="xhtml">PLG_LOADMODULE_FIELD_VALUE_DIVS</option>
-					<option value="rounded">PLG_LOADMODULE_FIELD_VALUE_MULTIPLEDIVS</option>
-					<option value="none">PLG_LOADMODULE_FIELD_VALUE_RAW</option>
+					default="table"
+					>
+					<option
+						value="table"
```
Comment (JoomliC) [inline plugins/content/loadmodule/loadmodule.xml]: I think you can keep option value on one line as only one attribute?

Comment (andrepereiradasilva) [inline plugins/content/loadmodule/loadmodule.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on b609d30cc6d886c81bc97ba47f532545565a72c5<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10472">issues.joomla.org/joomla-cms/10472</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on b609d30cc6d886c81bc97ba47f532545565a72c5<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10472">issues.joomla.org/joomla-cms/10472</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10472">issues.joomla.org/joomla-cms/10472</a>.</sub>

**Pull request merged**